cloud/kubernetes/helm/values.yaml

```diff
@@ -55,6 +55,11 @@ inferencenode:
   containers:
     name: inferencenode
 
+# Resource memory for EKS Fargate pods
+resources:
```

Comment:
@LakshmiMekala Can you find out optimum values which work for all providers so that - no need to maintain limits only for AWS Fargate.

Reply:
@rameshpolishetti as discussed for now we will stick resources to aws fargate only